### DISCUSSION TITLE
Solana

### DISCUSSION THREAD

**Discussion by GoldTaro** [Huniko-Team/community#192] [Q&A] [answered]
How does Solana achieve high scalability compared to other blockchains?

**Comment by Tvenus** [comment, marked as answer]
Solana achieves high scalability through several innovations including Proof of History, Tower BFT consensus, Turbine block propagation, Gulf Stream transaction forwarding, Sealevel parallel smart contracts runtime, Pipelining transaction processing, Cloudbreak horizontally scaled accounts database, and Archivers distributed ledger storage.

**Comment by GoldTaro** [comment]
Thank you
